groups/sig-testing/groups.yaml

```diff
@@ -191,14 +191,14 @@ groups:
   - email-id: k8s-infra-cherrypick-robot@kubernetes.io
     name: k8s-infra-cherrypick-robot
     description: |-
-      Github admins for github account k8s-cherry-pick-bot (to be renamed to match group name)
+      Github admins for github account k8s-infra-cherry-pick-bot.
+      2fa in kubernetes.1password.com SIG Testing vault
```

Comment:
@spiffxp We need to be sure the those lists have access to this vault. (I currently don't have access to this vault).